### PR TITLE
111/send mail bug

### LINKED ISSUE #111: Send EMails
- [x] Cant send registration email
- [x] Send reminder mail by admin or sfd
- [x] Send end of registration mail button

### PR DESCRIPTION
**Pull request by KordonDev**
closes #111

### CODE REVIEW

File: frontend/src/components/settings/MailSettings.vue [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              </v-btn>
+            </template>
+            <v-card>
+              <form v-on:submit.prevent="sendReminderEmail">
+                <v-card-title class="headline">
+                  Errinnerungsmail versenden
+                </v-card-title>
+                <v-card-text v-if="!emailSent">
+                  An welche Gruppe soll die Mail verschickt werden?
+                  <v-radio-group v-model="sentTo" required>
```
[the remark this answers is not in the document]
Comment (crymis) [reply frontend/src/components/settings/MailSettings.vue:29]: I'll check...

[the remark this answers is not in the document]
Comment (crymis) [reply frontend/src/components/settings/MailSettings.vue:29]: As I see it from the docs, there is no 'required' prop on the v-radio-group component.
What you can do instead is setting required to all the elements and by setting the same name to all radio-inputs, the browser knows that a selection is required, see also here: https://stackoverflow.com/questions/8287779/how-to-use-the-required-attribute-with-a-radio-input-field

File: frontend/src/components/settings/MailSettings.vue [hunk 2]
```diff
@@ -0,0 +1,191 @@
+<template>
+  <v-card class="mb-16">
+    <h2 class="ml-12">Mails</h2>
+    <v-row justify="center">
+      <v-col sm="12" md="12" lg="10" xl="8">
+        <h3>Errinnerungsmail</h3>
+        <MailReminderText />
+        <v-row justify="end">
+          <v-dialog v-model="reminderDialogOpen" persistent max-width="500">
```
[the remark this answers is not in the document]
Comment (crymis) [reply frontend/src/components/settings/MailSettings.vue:9]: Pretty strange. It looks to me that this issue comes from using v-card inside v-dialog, but in the vuetify examples of v-dialog they also use it that way... 
By setting margin-bottom to 0 on the v-card the issue is fixed here.

File: frontend/src/components/DateAndTime.vue [hunk 3]
```diff
@@ -10,7 +10,7 @@
         class="mx-4"
       />
     </v-col>
-    <v-col sm="8">
+    <v-col sm="6">
```
Comment (KordonDev) [inline frontend/src/components/DateAndTime.vue:13]: Are these changes correct?

Comment (KordonDev) [reply frontend/src/components/DateAndTime.vue:13]: looks good

**Pull request merged**